### PR TITLE
Add help desk page

### PR DESCRIPTION
**Pull request by Ludikrous**
Adds the help desk page, as per the [figma design](https://www.figma.com/file/UepzRD6mBR16n1C4aWtT6H/Core-Features?node-id=147%3A763).
Design is responsive!

The figma asks for an arrow to the intercom widget, but that seemed kinda tough. Instead, I put another button that'll open the intercom widget (works on mobile too).

helpdesk-web

image
image
image

### CODE REVIEW

**Comment by Ludikrous** [comment]
PR also adds some things to the global css file, like colors from the style guide

File: frontend/src/assets/global.css [hunk 1]
```diff
@@ -53,4 +66,13 @@ p.bold {
 }
 p.semi {
 	font-weight: 500;
+}
+
+a {
+	color: var(--pastel-purple);
+	text-decoration: underline;
```
Comment (burbelohugo) [inline frontend/src/assets/global.css:73]: I think this is adding an underline styling to the nav bar when it shouldn't, can you fix that please?
Screen Shot 2020-07-11 at 1 00 54 PM

File: frontend/src/views/HelpDesk.vue [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+            </p>
+          </div>
+      </div>
+      <div class="row my-3 my-md-5">
+        <div class="col-md-3"></div>
+        <div class="col-md-2 text-md-right">
+          <p class="btn filler pb-0" disabled="true"><b>Need help?</b></p>
+        </div>
+        <div class="col-md-7 text-md-left">
+          <div><a href="https://google.com" class="btn btn-purple"><b>Talk to an organizer over video</b></a></div>
```
Comment (burbelohugo) [inline frontend/src/views/HelpDesk.vue:23]: Can we use our re-usable button component here to ensure consistency?

File: frontend/src/views/HelpDesk.vue [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+    </div>
+  </div>
+</template>
+
+<script>
+import generalMixin from '../mixins/general';
+import Config from '../config/general';
+export default {
+  name: 'Help Desk',
+  props: {
```
Comment (burbelohugo) [inline frontend/src/views/HelpDesk.vue:36]: Is this prop used? If not, can we remove it?

File: frontend/src/views/HelpDesk.vue [hunk 4]
```diff
@@ -0,0 +1,79 @@
...
+    msg: String,
+  },
+  mixins: [generalMixin],
+  data(){
+    return {}
+  },
+  async mounted(){
+  },
+  methods: {
```
Comment (burbelohugo) [inline frontend/src/views/HelpDesk.vue:45]: Are the mounted, data, or methods resources used in this component? If not, can we remove them?

**Comment by burbelohugo** [comment]
The purple box seems to have more padding around the edges in comparison to the text location than how it looks in the mocks, can you make it tighter? That way there's less empty padding on the right and the left of the text

Screen Shot 2020-07-11 at 1 03 59 PM

File: frontend/src/views/HelpDesk.vue [hunk 5]
```diff
@@ -0,0 +1,79 @@
...
+          </div>
+      </div>
+      <div class="row my-3 my-md-5">
+        <div class="col-md-3"></div>
+        <div class="col-md-2 text-md-right">
+          <p class="btn filler pb-0" disabled="true"><b>Need help?</b></p>
+        </div>
+        <div class="col-md-7 text-md-left">
+          <div><a href="https://google.com" class="btn btn-purple"><b>Talk to an organizer over video</b></a></div>
+          <div class="mt-2"><p class="btn btn-purple" onclick="Intercom('show')"><b>Or, chat with an organizer here</b></p></div>
```
Comment (burbelohugo) [inline frontend/src/views/HelpDesk.vue:24]: This works so well! Great idea

**Comment by burbelohugo** [comment]
@Ludikrous amazing work on this 💯

**Pull request merged**